code/_core/obj/item/dying.dm

```diff
@@ -15,14 +15,29 @@
 		return FALSE
 
 	var/choice
-
+	var/mob/E = caller
 	if(length(polymorphs))
+		if(E.attack_flags & CONTROL_MOD_ALT && D.type == /obj/item/slime_core/custom)
```

Comment:
You should probably add the copy+paste feature to the slime core code instead of here.